docs/specification/well-known-schema-formats.md

```diff
@@ -0,0 +1,33 @@
+## Well-known schema formats
+
+### proto
+
+- schema: .proto
+- schema_name: unspecified
+
+### proto-set
+
+- schema: [FileDescriptorSet](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/descriptor.proto)
+- schema_name: Fully qualified name to the message within the descriptor set.
+
+A DescriptorSet for the following .proto file, the fully qualified name for `Baz` is `foo.bar.Baz`.
```

Comment:
This sentence seems confusing. We probably need discussion of how to generate the binary descriptor set (e.g. `protoc --descriptor_set_out`). See also https://foxglove.dev/docs/connection/foxglove-websocket